flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/CheckpointBarrierAlignerTestBase.java

```diff
@@ -369,9 +369,7 @@ public void testMissingCancellationBarriers() throws Exception {
 		inputGate = createBarrierBuffer(2, sequence, validator);
```

Comment:
Why has this test and `CheckpointBarrierTrackerTest.java` changed in this commit? Rebasing/squashing mistake, or am I missing something about this commit (I thought it's a pure clean up without functional changes).